### PR TITLE
ISSUE-646 Replace VectorDrawable with MaterialIcon in main

### LINKED ISSUE #646: Use compose material icons instead of VectorDrawable
## Kind (Required)

- Improvement

 ## Overview (Required)

- I think we can use it's icons 😄 
- https://developer.android.com/reference/kotlin/androidx/compose/material/icons/package-summary?hl=ja
- https://github.com/DroidKaigi/conference-app-2021/tree/main/uicomponent-compose/main/src/main/res/drawable

 ## Links	

-

### PR DESCRIPTION
**Pull request by kuramu1108**
## Issue
- close #646

## Overview (Required)
- Remove vector resources in https://github.com/DroidKaigi/conference-app-2021/tree/main/uicomponent-compose/main/src/main/res/drawable
- No effective changes UI-wise

## Links
- NA

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Review by takahirom — APPROVED** [review]
🆒

File: uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/DrawerContent.kt [hunk 1]
```diff
@@ -35,62 +46,61 @@ import io.github.droidkaigi.feeder.timetable2021.TimetableTab
 
 enum class DrawerContents(
     val group: Group,
-    @DrawableRes
```
Comment (takahirom) [inline uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/DrawerContent.kt:38]: Can you remove this import? 🙏 

```
> The following files had format violations:
      uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/DrawerContent.kt
Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.
          @@ -1,6 +1,5 @@

           package·io.github.droidkaigi.feeder
You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.
           

          -import·androidx.annotation.DrawableRes
See https://docs.gradle.org/7.2/userguide/command_line_interface.html#sec:command_line_warnings
           import·androidx.compose.foundation.Image
8 actionable tasks: 1 executed, 7 up-to-date
           import·androidx.compose.foundation.layout.Arrangement
           import·androidx.compose.foundation.layout.Column
  Run './gradlew :spotlessApply' to fix these violations.
```

Comment (kuramu1108) [reply uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/DrawerContent.kt:38]: oops missed it, 4a90e65 🙇

**Comment by takahirom** [comment]
Thank you!

**Pull request merged**